ovos_config/__init__.py

```diff
@@ -3,4 +3,4 @@
     set_default_tz, set_default_lang, get_default_tz, get_default_lang, \
     get_config_tz, get_primary_lang_code, load_languages, load_language
 from ovos_config.locations import SYSTEM_CONFIG, USER_CONFIG, DEFAULT_CONFIG, get_xdg_config_locations
-from ovos_config.ovos import get_ovos_config
+from ovos_config.meta_config import get_ovos_config
```

Comment:
Can we name this just meta? Avoid repeating words in the import

Reply:
sure